assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -197,13 +209,7 @@ function Title({
         aria-controls={panelContentId}
       >
         <IconWrapper>{canCollapse && toggleIcon}</IconWrapper>
-        <Heading
-          id={panelTitleId}
-          as="span"
```

Comment:
`span` isn't an available option on the heading. `as` was changing this to a `span` and the `size` wasn't getting applied correctly. Moved the heading up and made this a `span` to fix the styling

Reply:
would it work to just make `Heading` have `display: inline` so that it's treated like a span so that the recreation of the styling doesn't have to exist? 

Then again, this is the pattern that exists in the panels and if we want to keep it that way it's fine. Just seems a little odd to have `h2 > button > div > span`.

Reply:
I agree it's a little strange. I can add a tech debt ticket to come back and look at this